tests/gpflow/experimental/check_shapes/test_check_shapes.py

```diff
@@ -382,33 +394,65 @@ def h(a: TensorType, b: TensorType) -> TensorType:
             "b: [d...]",
             "return: [d...]",
         )
-        def f(a: TensorType, b: TensorType) -> TensorType:
+        def f(a: TestShaped, b: TestShaped) -> TestShaped:
             return a
 
         f(t(2, 3), t(2, 4))  # Wrong shape, but checks disabled.
 
     f(t(2, 3), t(2, 4))  # Wrong shape, but checks were disable when function was created.
-    get_check_shapes(f)(h)  # Don't crash.
+    get_check_shapes(f)(h)  # pylint: disable=not-callable  # Don't crash.
 
     @check_shapes(
         "a: [d...]",
         "b: [d...]",
         "return: [d...]",
     )
-    def g(a: TensorType, b: TensorType) -> TensorType:
+    def g(a: TestShaped, b: TestShaped) -> TestShaped:
         return a
 
     with pytest.raises(ShapeMismatchError):
         g(t(2, 3), t(2, 4))
-        get_check_shapes(g)(h)  # Don't crash.
+        get_check_shapes(g)(h)  # pylint: disable=not-callable    # Don't crash.
```

Comment:
(out of curiosity: any idea why pylint complains about this?)

Reply:
> (out of curiosity: any idea why pylint complains about this?)

Hmm, it doesn't seem to do it any more. Maybe I had a buggy version of `pylint`? It does seem like a weird error.